src/arguments.rs

```diff
@@ -1,20 +1,40 @@
-use crate::common::*;
+use crate::walker::{Walker, WalkerOptions};
+use present::{File, Result};
```

Comment:
Small nitpick:

```suggestion
use {
  crate::walker::{Walker, WalkerOptions}, 
  present::{File, Result},
  clap::Parser as StructOpt
};
```

repeat for all files

Reply:
Gotta keep things clean, I'm on it!